<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roomListForm</class>
 <widget class="QWidget" name="roomListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_room">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_mode">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>音</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_num">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1/6人</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_method">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>xx玩法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_joinRoom">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>·</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_state">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>准备阶段</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_level">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LV.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72, 84, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_roomid">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号：123456</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pb_room</zorder>
  <zorder>pb_mode</zorder>
  <zorder>lb_num</zorder>
  <zorder>lb_method</zorder>
  <zorder>pb_joinRoom</zorder>
  <zorder>pushButton</zorder>
  <zorder>lb_state</zorder>
  <zorder>lb_level</zorder>
  <zorder>lb_roomid</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
